oauth2/src/main/java/com/okta/spring/boot/oauth/OktaOAuth2Configurer.java

```diff
@@ -159,6 +185,11 @@ private void configureResourceServerForJwtValidation(HttpSecurity http, OktaOAut
     private void configureResourceServerForOpaqueTokenValidation(HttpSecurity http, OktaOAuth2Properties oktaOAuth2Properties) throws Exception {
 
         if (!isEmpty(oktaOAuth2Properties.getClientId()) && !isEmpty(oktaOAuth2Properties.getClientSecret())) {
+            // Spring (2.7.x+) configures JWT be default and this creates startup failure "Spring Security
+            // only supports JWTs or Opaque Tokens, not both at the same time" when we try to configure Opaque Token mode in following line.
+            // Therefore, we are unsetting JWT mode before attempting to configure Opaque Token mode for ROOT issuer case.
+            unsetJwtConfigurer(http.getConfigurer(OAuth2ResourceServerConfigurer.class));
+
```

Comment:
It's possible this logic (and the related reflection) could be removed with some updates to the
Env property mapper:
https://github.com/okta/okta-spring-boot/blob/master/oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java#L177-L185

Reply:
Yeah, will plan to remove this in near future as a separate task and not part of this PR.